src/promptflow-core/promptflow/core/_prompty_utils.py

```diff
@@ -240,7 +253,14 @@ def format_stream(llm_response):
     if api == "completion":
         result = format_choice(response.choices[0].text)
     else:
-        result = format_choice(getattr(response.choices[0].message, "content", ""))
+        # When calling function/tool, function_call/tool_call response will be returned as a field in message,
+        # so we need return message directly. Otherwise, we only return content.
```

Comment:
waht's the behavior of our chat ui when seeing the tool output?

Reply:
![image](https://github.com/microsoft/promptflow/assets/17938940/c2dfb3b2-dae2-4d2a-a3fd-81d65a965cdd)
it will not show the tool output
tool output will show in trace
![image](https://github.com/microsoft/promptflow/assets/17938940/229df022-6e85-4f83-8333-31bc9aefebb9)

Reply:
Shall ask UI devs on support of this.

Reply:
Treat this as a test example?